src/applications/personalization/profile360/vet360/tests/selectors.unit.spec.js

```diff
@@ -55,7 +55,11 @@ describe('selectors', () => {
         'returns true when on localhost so the local mock Vet360 will run',
       ).to.be.true;
 
-      global.document.location.hostname = 'staging.vets.gov';
+      /* eslint-disable no-undef */
+      jsdom.reconfigure({
```

Comment:
It's recommended to use [`jest-environment-jsdom-global`](https://www.npmjs.com/package/jest-environment-jsdom-global) for this, according to `https://github.com/facebook/jest/issues/5124#issuecomment-352838312`.